API Fortress Integration with DataDog

jioanni opened the pull request:
Attached, please find the PR for API Fortress' integration with DataDog. 

Best,

Jason

jioanni committed:
fixed

dsahni reviewed (changes requested):
Thanks for your PR @jioanni - could you please add documentation about the integration in a Readme.md (making sure to follow the heading format outlines [here](https://github.com/DataDog/IntegrationTemplate/blob/master/integration/README.md)) and also including the metadata.csv file with any metrics the integration sends to Datadog?

jioanni committed:
readme

jioanni committed:
csv

jioanni committed:
fixed doc link

jioanni commented:
Weird; Readme was originally in there. Fixed, added CSV, not super clear on the documentation for said CSV. The link in the original readme redirects when clicked to unrelated documentation. Let me know what else it needs, if anything. 

- Jason

jioanni committed:
metadata fix

jioanni commented:
Updated metadata.csv
